### PR TITLE
Initial server quick start guide docs added.

### PR DESCRIPTION
**Pull request by kasenvr**

### CODE REVIEW

File: docs/source/host/quick-start.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+########################
+
+.. contents:: On This Page
+    :depth: 2
+    
+----------------------------
+Notice: Port Forwarding
+----------------------------
+
+There are two ways to make a domain accessible from the outside internet.
```
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:12]:
```suggestion
There are two ways to make a domain accessible from the outside Internet.
```

File: docs/source/host/quick-start.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+2. Alternatively, you can manually open the following ports on your router and firewall:
+
+    * 40100 : (tcp) administration web portal
+    * 40102 : (udp) main connection for clients
+
+----------------------------
+Windows
+----------------------------
+
+1. Download and Vircadia (full, not Interface only).
```
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:24]:
```suggestion
1. Download Vircadia (full, not Interface only).
```

File: docs/source/host/quick-start.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked.
+
+.. image:: _images/full-install-components.png
+
+3. In your start menu, search for and run `Vircadia Sandbox`.
+4. Go to ``http://localhost:40100/`` (if the server is on a different machine, then replace 'localhost' with the IP address) in your web browser and complete the wizard to set up your domain.
```
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:30]:
```suggestion
4. Go to ``http://localhost:40100/`` in your web browser and complete the wizard to set up your domain. If the server is on a different machine, then replace 'localhost' with its IP address.
```

File: docs/source/host/quick-start.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+5. In your Vircadia Interface, open the `Explore` app and enter `localhost` or your IP address to get there.
```
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:31]:
```suggestion
5. In your Vircadia Interface, open the `Explore` app and enter `localhost` (or the server's IP address) to get there.
```

File: docs/source/host/quick-start.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    * 40100 : (tcp) administration web portal
+    * 40102 : (udp) main connection for clients
+
+----------------------------
+Windows
+----------------------------
+
+1. Download and Vircadia (full, not Interface only).
+2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked.
```
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:25]:
```suggestion
2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked. Vircadia Sandbox is the domain server.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/host/quick-start.rst (hunk 1 above)
Comment (kasenvr) [reply docs/source/host/quick-start.rst:12]: This is a bit of an odd one, personally I wouldn't capitalize it, I don't capitalize "metaverse" either typically but apparently in America and academic forums in America they do capitalize it, so what the heck. lol 
https://en.wikipedia.org/wiki/Capitalization_of_Internet 
https://grammarist.com/style/internet-capitalization/

**Review by ctrlaltdavid — APPROVED** [review]

File: docs/source/host/quick-start.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+2. Alternatively, you can manually open the following ports on your router and firewall:
+
+    * 40100 : (tcp) administration web portal
+    * 40102 : (udp) main connection for clients
+
+----------------------------
+Windows
+----------------------------
+
+1. Download Vircadia (full, not Interface only).
```
Comment (JulianGro) [inline docs/source/host/quick-start.rst:24]:
```suggestion
1. Download the Vircadia Installer.
```
I don't see an Interface only installer on the website; Only the launcher.

File: docs/source/host/quick-start.rst [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked. Vircadia Sandbox is the domain server.
```
Comment (JulianGro) [inline docs/source/host/quick-start.rst:25]: Has this not been renamed to Vircadia Server yet? (I am not on Windows so I wouldn't know)

File: docs/source/host/quick-start.rst [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+.. image:: _images/full-install-components.png
+
+3. In your start menu, search for and run `Vircadia Sandbox`.
+4. Go to ``http://localhost:40100/`` in your web browser and complete the wizard to set up your domain. If the server is on a different machine, then replace 'localhost' with its IP address.
+5. In your Vircadia Interface, open the `Explore` app and enter `localhost` (or the server's IP address) to get there.
```
Comment (JulianGro) [inline docs/source/host/quick-start.rst:31]:
```suggestion
5. In your Vircadia Interface, open the `Explore` app and enter `localhost` (or the server's address) to get there.
```
Same as above: You can also use hostnames like vircadia.com or localhost.

**Review by JulianGro — CHANGES_REQUESTED** [review]

**Comment by digisomni** [comment]
I need to update the image to the latest (showing Vircadia Server).

**Comment by digisomni** [comment]
Will wait for PR #138 to be merged first before addressing and merging this.

**Comment by JulianGro** [comment]
Could you rebase this on master?

**Pull request closed without merging**